flict/impl.py

```diff
@@ -3,137 +3,109 @@
 # flict - FOSS License Compatibility Tool
 #
 # SPDX-FileCopyrightText: 2021 Jens Erdmann
+# SPDX-FileCopyrightText: 2022 Henrik Sandklef
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 #
 ###################################################################
 
 
-from flict.flictlib.compatibility import Compatibility
-from flict.flictlib.format.factory import FormatFactory
-from flict.flictlib.license import LicenseHandler, decode_license_expression, encode_license_expression
-from flict.flictlib.policy import Policy
-from flict.flictlib.project import Project
-from flict.flictlib.report import Report
 from flict.flictlib.return_codes import FlictError, ReturnCodes
 
+from flict.flictlib.arbiter import Arbiter
+from flict.flictlib.format.factory import FormatterFactory
+from flict.flictlib.project.reader import ProjectReaderFactory
+
 import json
 
 
 class FlictImpl:
+
     def __init__(self, args) -> None:
         self._args = args
-        self._formatter = FormatFactory.formatter(args.output_format)
-        self._license_handler = LicenseHandler(args.translations_file, args.relicense_file, "")
-        self._compatibility = Compatibility(args.matrix_file,
-                                            args.extended_licenses)
+        self._formatter = FormatterFactory.formatter(args.output_format)
 
-    def _empty_project_report(self, licenses):
-        project = Project(None, self._license_handler, licenses)
-        return Report(project, self._compatibility)
+    def merge_license_db(self):
+        arbiter = self._get_arbiter()
```

Comment:
If every method needs access to an arbiter, an arbiter should be part of `__init__`